staging/src/k8s.io/apiserver/pkg/registry/rest/update.go

```diff
@@ -51,6 +52,19 @@ type RESTUpdateStrategy interface {
 	// filled in before the object is persisted.  This method should not mutate
 	// the object.
 	ValidateUpdate(ctx context.Context, obj, old runtime.Object) field.ErrorList
+	// WarningsOnUpdate returns any warnings that should be returned to the updater.
```

Comment:
Reads a bit wierdly.  `WarningsOnUpdate returns warnings to the client performing an update.`